src/libs/actions/Report.js

```diff
@@ -962,8 +968,8 @@ function fetchAllReports(
                 // content and improve chat switching experience by only downloading content we don't have yet.
                 // This improves performance significantly when reconnecting by limiting API requests and unnecessary
                 // data processing by Onyx.
-                const reportIDsWithMissingActions = _.filter(returnedReportIDs, id => (
-                    isReportMissingActions(id, reportMaxSequenceNumbers[id])
+                const reportIDsWithMissingActions = _.filter(returnedReports, report => (
+                    isReportMissingActions(report.reportID, reportMaxSequenceNumbers[report.reportID])
```

Comment:
This is a 🐛  because we are expecting `reportIDsWithMissingActions` to be an array of `reportIDs`. But we're now filtering an array of objects.

Reply:
Thanks for catching that!